core/templates/domain/skill/skill-backend-api.service.spec.ts

```diff
@@ -44,7 +43,6 @@ describe('Skill backend API service', function() {
   beforeEach(angular.mock.inject(function($injector, $q) {
     SkillBackendApiService = $injector.get(
       'SkillBackendApiService');
-    UndoRedoService = $injector.get('UndoRedoService');
```

Comment:
(Again) Looks like you don't need the undo-redo service. Why haven't you removed this from the require? (Ditto for all other test files)

Reply:
Done.